install/cluster-operator/040-Crd-kafka.yaml

```diff
@@ -33,6 +33,8 @@ spec:
     JSONPath: .spec.zookeeper.replicas
     type: integer
     priority: 0
+  subresources:
```

Comment:
You have to run `make crd_install` to propagate the CRD YAML changes to all directories.